mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/55 - objectives UI

Closes #55 

To test:
- Run introductory scenario
- Click Objectives tab to see current objectives
- Make sure help tip UI acknowledges objectives UI has been opened
- Check objectives tab as objectives/phases get completed to make sure it updates properly

Notes:
- There is currently a bug with the server sending the wrong phase being completed (it sends the new active phase instead), but Mike has been notified and will hopefully fix this
